packages/clangml/clangml.3.5.0/files/clangml.install

```diff
@@ -0,0 +1,3 @@
+bin: [
+  "_build/consumer/processor.native" {"clangml-processor"}
+]
```

Comment:
OK, I got you. For the next version I will look into it. But now I will stick to the tarballs we already have.